sys/linux/socket_netlink_route.txt

```diff
@@ -21,6 +21,11 @@ include <uapi/linux/net_namespace.h>
 include <uapi/linux/veth.h>
 include <uapi/linux/can/gw.h>
 include <uapi/linux/can/vxcan.h>
+include <uapi/linux/mpls_iptunnel.h>
+include <uapi/linux/ila.h>
+#include <uapi/linux/seg6_iptunnel.h>
```

Comment:
Not needed? Remove if so.

Reply:
It is needed for SEG6_ENCAP. I thought it could've been uncommented along with the structures

Reply:
Ah, OK, got it.